lib/src/main/java/com/yalantis/contextmenu/lib/Utils.kt

```diff
@@ -0,0 +1,141 @@
+package com.yalantis.contextmenu.lib
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Build
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.yalantis.contextmenu.lib.extensions.getColorCompat
+import com.yalantis.contextmenu.lib.extensions.getDimension
+
+object Utils {
```

Comment:
Let it be a file with Context extensions, not an object. Try to think in a more Kotlin way.